apps/discovery_api/test/unit/discovery_api/event_handler_test.exs

```diff
@@ -65,4 +68,20 @@ defmodule DiscoveryApi.EventHandlerTest do
       assert_called(DataJsonService.delete_data_json())
     end
   end
+
+  describe "handle_event/1 #{dataset_delete()}" do
```

Comment:
Is there value in testing the individual parts of this process? Should we at least `expect` these calls with the appropriate parameters?